src/Toolchain/ToolchainProvider.ts

```diff
@@ -188,21 +188,28 @@ export class ToolchainProvider implements vscode.TreeDataProvider<BaseNode> {
 
   uninstall(tnode: ToolchainNode) {
     if (tnode === undefined) {
-      throw Error('Invalid toolchain node');
+      throw Error('Invalid toolchain node.');
     }
 
     const backendName = tnode.backendName;
     if (backendName === undefined) {
-      throw Error('Invalid toolchain node');
+      throw Error('Invalid toolchain node.');
     }
 
-    gToolchainEnvMap[backendName].uninstall(tnode.toolchain).then(() => {
-      if (DefaultToolchain.getInstance().isEqual(tnode.toolchain)) {
-        Logger.info(this.tag, 'Default toolchain is cancelled.');
-        DefaultToolchain.getInstance().unset();
-      }
-      this.refresh();
-    });
+    gToolchainEnvMap[backendName]
+        .uninstall(tnode.toolchain)
+        .then(
+            () => {
+              vscode.window.showInformationMessage(`Uninstallation was successful.`);
```

Comment:
I think I heard below expression more, so I suggest: 

```suggestion
              vscode.window.showInformationMessage(`Successfully uninstalled.`);
```